_includes/about.html

```diff
@@ -10,5 +10,10 @@
       across the world who can watch, participate and share their views
       right from the comfort of their homes.
     </div>
+    <div class="content">
+      <a href="{{ 'about.html' | relative_url }}" class="primary-button">
```

Comment:
Something I missed reviewing earlier:

```suggestion
      <a href="{{ '/about' | relative_url }}" class="primary-button">
```

The `Learn More` link is going to a `.html` page which doesn't look much good URL wise.

Reply:
I was going to do just about. But, I saw the coc page was html, so I added it.